site_persistence_file.json

```diff
@@ -160,30 +148,57 @@
           "value": "wisercare"
         }
       ], 
-      "name": "UCSF Medical Center", 
+      "name": "University of California, San Francisco", 
       "resourceType": "Organization", 
       "telecom": []
     }, 
     {
       "address": [], 
-      "id": 301, 
+      "id": 201, 
       "identifier": [
         {
           "assigner": null, 
           "system": "http://hl7.org/fhir/sid/us-npi", 
           "use": "official", 
-          "value": "1639429368"
+          "value": "TEMP201"
         }, 
```

Comment:
Just remove the us-npi if it's not legit, I'd say.

Reply:
Will do tomorrow

Reply:
I believe you're waiting on Lee to commit changes.  Once this is ready to merge, I'll take on the task of moving the site_persistence files within the main repo.